### PR TITLE
[IMP] l10n_de_tax_statement : add chatter in form view

### PR DESCRIPTION
**Pull request by chandni299**
[IMP] l10n_de_tax_statement : add chatter in form view

### CODE REVIEW

**Review by marylla — APPROVED** [review]
Looks good to me. :+1:

File: l10n_de_tax_statement/views/l10n_de_tax_statement_view.xml [hunk 1]
```diff
@@ -104,6 +104,11 @@
                         </page>
                     </notebook>
                 </sheet>
+                <div class="oe_chatter">
+                    <field name="message_follower_ids" widget="mail_followers"/>
+                    <field name="message_ids" widget="mail_thread"/>
+                    <field name="activity_ids" widget="mail_activity"></field>
```
Comment (yweng8111) [inline l10n_de_tax_statement/views/l10n_de_tax_statement_view.xml]: It looks better, if you replace this line with this ;)
`<field name="activity_ids" widget="mail_activity"/>`

Comment (chandni299) [reply l10n_de_tax_statement/views/l10n_de_tax_statement_view.xml]: Done.

**Review by yweng8111 — APPROVED** [review]

**Comment by marylla** [comment]
@tv-openbig Hi Thorsten, Chandni added the chatter and activities to the Umsatzsteuervoranmeldung form view, so that we can save activities, for example to get reminded to pay the tax on a specific date. Can you review and merge?

**Comment by marylla** [comment]
@chandni299 Can you squash your commits?

**Comment by chandni299** [comment]
> @chandni299 Can you squash your commits?

I have done it already.

**Comment by tv-openbig** [comment]
@marylla @yweng8111 @chandni299
Bildschirmfoto 2021-10-21 um 17 33 15

Nice additional improvement. It is definitely usefull.

**Pull request merged**